ios/Classes/MapboxMapController.swift

```diff
@@ -483,6 +471,7 @@ class MapboxMapController: NSObject, FlutterPlatformView, MGLMapViewDelegate, Ma
             
             for line in lineAnnotationController.styleAnnotations() {
                 if line.identifier == lineId {
+                    Convert.interpretGeometryUpdate(options: arguments["options"], delegate: line as! MGLLineStyleAnnotation)
```

Comment:
`interpretLineUpdate` insed of `interpretGeometryUpdate`?

Reply:
Same though here, but we can rename the method if it can be reused later.